substratevm/src/com.oracle.graal.pointsto/src/com/oracle/graal/pointsto/reports/CallTreePrinter.java

```diff
@@ -166,29 +164,29 @@ public void buildCallTree() {
              * Process the method: iterate the invokes, for each invoke iterate the callees, if the
              * callee was not already processed add it to the tree and to the work list.
              */
-            node.method.getTypeFlow().getInvokes().stream()
-                            .sorted(invokeComparator)
-                            .forEach(invoke -> processInvoke(invoke, node, workList));
-
+            // todo test nothing has been broken here
```

Comment:
Please make sure there are no TODOs left behind. Why is it necessary to introduce `AnalysisMethod.getCallees()`?